piptools/sync.py

```diff
@@ -138,7 +138,7 @@ def sync(to_install, to_uninstall, verbose=False, dry_run=False, pip_flags=None,
         pip = 'pip'
     else:
         # find pip in same directory as pip-sync entry-point script
-        pip = os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])), 'pip')
+        pip = os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])), 'pip%s' % sys.version_info[:1])
```

Comment:
Not sure if this is guruanteed to exist always.  It should fallback to just "pip" maybe if it does not exist.

Reply:
Also, what about using `[sys.executable, "-m", "pip"]` ?

Reply:
This could be used in general, also with `$VIRTUAL_ENV`.

Reply:
Oh, that's a beautiful suggestion!

Reply:
Fixed!